### PR TITLE
200 prepare release 10220

### LINKED ISSUE #200: Prepare Release 10.2.2.0
#### Checklist (after the release has been done on the `khiops` repository; see https://github.com/KhiopsML/khiops/issues/322):
- [x] Create branch
- [x] Update CHANGELOG.md
- [x] Update `DEFAULT_KHIOPS_REVISION` in [.github/workflows/dev-docker.yml] to 10.2.2
- [x] Update `DEFAULT_KHIOPS_CORE_VERSION` in [.github/workflows/conda.yml] to 10.2.2
- [x] Push this issue branch and create PR for it
- [x] Launch [.github/workflows/dev-docker.yml] manually to update the development Docker images to the 10.2.2 Khiops core packages; make sure that:
   - Khiops core version is set to 10.2.2
   - The choice to push the images to the GitHub registry is checked. 
- [ ] Merge PR into `dev`
- [ ] Create PR to merge `dev` into `main`;
- [ ] Merge PR into `main`;
- [ ] Git tag `main` with 10.2.2.0;
- [ ] Push the 10.2.2.0 tag on `main`;
- [ ] Launch [.github/workflows/pip.yml] and [.github/workflows/conda.yml] manually, on the 10.2.2.0 tag; thusly, the Conda packages are pushed on the default `khiops-dev` channel;
- [ ] Launch [.github/workflows/conda.yml] manually, on the 10.2.2.0 tag, selecting the official `khiops` channel;
- [ ] Copy relevant CHANGELOG.md contents to the 10.2.2.0 GitHub pre-release;
- [ ] Launch [.github/workflows/api-docs.yml] manually on the 10.2.2.0 tag and instruct it to deploy the documentation pages;
- [ ] Transform the 10.2.2.0 GitHub pre-release into a release.

### PR DESCRIPTION
**Pull request by popescu-v**
closes #200

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,22 @@
...
+- (`sklearn`) Sparse array inputs to sklearn estimator methods are now supported.
+
+### Changed
+- *Internals*:
+  - OpenMPI is now used for native + Pip-based Linux installations.
+  - Pre-release versions with dots in the pre-release token are now accepted.
+
+### Fixed
+- `core`
+  - Metric name search in estimator analyis report is now correct.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:21]: Eliminate "is now correct".

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,22 @@
...
+  - Interval partition minimum size in DataGridDimension check in estimator
+    analysis report is now set to 1.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:23]: Not worth to report.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,22 @@
...
 - Internals: Changes in *Internals* sections are unlikely to be of interest for data scientists.
 
+## 10.2.2.0 - 2024-07-03
+
+### Added
+- (`sklearn`) Sparse array inputs to sklearn estimator methods are now supported.
+
+### Changed
+- *Internals*:
+  - OpenMPI is now used for native + Pip-based Linux installations.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:16]: `MPI backend from MPICH to OpenMPI for native ...`

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,22 @@
...
+  - Pre-release versions with dots in the pre-release token are now accepted.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:17]: Not worth reporting.

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,22 @@
   - Example: 10.2.1.4 is the 5th version that supports khiops 10.2.1.
 - Internals: Changes in *Internals* sections are unlikely to be of interest for data scientists.
 
+## 10.2.2.0 - 2024-07-03
+
+### Added
+- (`sklearn`) Sparse array inputs to sklearn estimator methods are now supported.
```
Comment (folmos-at-orange) [inline CHANGELOG.md:12]: `Support for sparse arrays in sklearn estimators`

File: .github/workflows/api-docs.yml [hunk 6]
```diff
@@ -68,7 +68,7 @@ jobs:
           path: doc/_build/html/
   # Deploy only when the user explicitly (and manually) orders it
   deploy:
-    if: ${{ github.event_name == 'workflow_dispatch' || inputs.deploy-gh-pages == true }}
+    if: ${{ github.event_name == 'workflow_dispatch' && inputs.deploy-gh-pages == true }}
```
Comment (folmos-at-orange) [inline .github/workflows/api-docs.yml:71]: Eliminate the `${{ }}` operator

**Review by folmos-at-orange — APPROVED** [review]
You can merge after the corrections.

File: CHANGELOG.md (hunk 1 above)
Comment (popescu-v) [reply CHANGELOG.md:21]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (popescu-v) [reply CHANGELOG.md:23]: Done: removed this paragraph.

File: CHANGELOG.md (hunk 3 above)
Comment (popescu-v) [reply CHANGELOG.md:16]: Ok. Done.

File: CHANGELOG.md (hunk 4 above)
Comment (popescu-v) [reply CHANGELOG.md:17]: Done: removed item.

File: CHANGELOG.md (hunk 5 above)
Comment (popescu-v) [reply CHANGELOG.md:12]: Ok, done.

File: .github/workflows/api-docs.yml (hunk 6 above)
Comment (popescu-v) [reply .github/workflows/api-docs.yml:71]: Done.

**Pull request merged**